documentation/book/assembly-setting-up-tracing-mirror-maker-connect.adoc

```diff
@@ -0,0 +1,11 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-tracing.adoc
+[id='assembly-setting-up-tracing-mirror-maker-connect-{context}']
+= Setting up tracing for Mirror Maker and Kafka Connect
+
+Distributed tracing is supported for Mirror Maker and Kafka Connect (including Kafka Connect with Source2Image support). 
+
+Only messages produced and consumed by Kafka Connect itself are traced. To trace messages sent between Kafka Connect and external systems, you must configure tracing in the connectors for those systems.
```

Comment:
Should we mention what Mirror Maker does as well?
A question for @scholzj ... does Mirror Maker adds any trace information when it consumes message from the source cluster and produces to the target one, right? Is it using interceptors, right?

Reply:
^ Question for @scholzj 

Sure, I will add a brief description of Mirror Maker here.

Reply:
Yes, you can see the message enter (consumer) and leave (producer) Mirror Maker in the traces. It basically adds the same as any other client.